### ISSUE TITLE
Put code into submodules

### ISSUE THREAD

**Issue by laperss** [laperss/xerces_cloud#16]
So that we can easily clone parts of the code: 

https://git-scm.com/docs/git-submodule

**Comment by ghost** [comment]
I agree that this is a good solution for software packages for which we want full control over the version. In the case of the submodule you added now, shouldn't this just be part of our main repository?

**Comment by laperss** [comment]
Yes I know, I just wanted for now a repo that I could clone to my devVM without getting everything on there. Also since it is a fork it was simpler to make it separate. We can change this later.

**Comment by ghost** [comment]
Ok great :)
